Karma tests failed in Chrome Headless

__Describe the Bug__

Tests failed in Chrome Headless with following error.

```
Chrome Headless 85.0.4182.0 (Windows 10) features/move - MovePreview markers should clone markers FAILED
        AssertionError: expected 'url("#marker-start-clone")' to deeply equal 'url(#marker-start-clone)'
            at eval (webpack-internal:///./test/spec/features/move/MovePreviewSpec.js:601:56)
            at Diagram.invoke (webpack-internal:///./node_modules/didi/dist/index.esm.js:196:20)
            at Context.eval (webpack-internal:///./test/helper/index.js:132:23)
Chrome Headless 85.0.4182.0 (Windows 10): Executed 1593 of 1598 (1 FAILED) (skipped 5) (17.37 secs / 11.666 secs)
TOTAL: 1 FAILED, 1592 SUCCESS
```
__Steps to Reproduce__

Steps to reproduce the behavior:

1. Run tests in ChromeHeadless `TEST_BROWSERS=ChromeHeadless npm run test`

__Expected Behavior__

All tests succeed


__Environment__

Please complete the following information:

 - Browser: Chrome Headless 85.0.4182.0
 - OS: Windows 10
 - Library version: develop

--- a/test/spec/features/move/MovePreviewSpec.js
+++ b/test/spec/features/move/MovePreviewSpec.js
@@ -598,18 +598,18 @@ describe('features/move - MovePreview', function() {
 
       var connection1Polyline = domQuery('[data-element-id="connection1"] polyline', dragGroup);
 
-      expect(connection1Polyline.style.markerStart).to.eql(idToReference(markerStartClone.id));
-      expect(connection1Polyline.style.markerEnd).to.eql(idToReference(markerEndClone.id));
+      expect(idToReferenceFormatOptions(markerStartClone.id)).to.deep.include(connection1Polyline.style.markerStart);
+      expect(idToReferenceFormatOptions(markerEndClone.id)).to.deep.include(connection1Polyline.style.markerEnd);
 
       var connection2Polyline = domQuery('[data-element-id="connection2"] polyline', dragGroup);
 
-      expect(connection2Polyline.style.markerStart).to.eql(idToReference(markerStartClone.id));
-      expect(connection2Polyline.style.markerMid).to.eql(idToReference(markerMidClone.id));
+      expect(idToReferenceFormatOptions(markerStartClone.id)).to.deep.include(connection2Polyline.style.markerStart);
+      expect(idToReferenceFormatOptions(markerMidClone.id)).to.deep.include(connection2Polyline.style.markerMid);
 
       var connection3Polyline = domQuery('[data-element-id="connection3"] polyline', dragGroup);
 
-      expect(connection3Polyline.style.markerMid).to.eql(idToReference(markerMidClone.id));
-      expect(connection3Polyline.style.markerEnd).to.eql(idToReference(markerEndClone.id));
+      expect(idToReferenceFormatOptions(markerMidClone.id)).to.deep.include(connection3Polyline.style.markerMid);
+      expect(idToReferenceFormatOptions(markerEndClone.id)).to.deep.include(connection3Polyline.style.markerEnd);
     }));
 
 
@@ -640,9 +640,27 @@ describe('features/move - MovePreview', function() {
  * Get functional IRI reference for given ID of fragment within current document.
  *
  * @param {string} id
+ * @param {string} [quoteSymbol='']
  *
  * @returns {string}
  */
-function idToReference(id) {
-  return 'url(#' + id + ')';
+function idToReference(id, quoteSymbol) {
+  quoteSymbol = quoteSymbol ? quoteSymbol : '';
+  return 'url(' + quoteSymbol + '#' + id + quoteSymbol + ')';
+}
+
+/**
+ * Get functional IRI reference in all syntactical valid options according to W3C.
+ * (See https://www.w3.org/TR/CSS2/syndata.html#uri). This will include one
+ * unquoted and two quoted (<'> and <">) variants and allows for testing on
+ * different browser environments.
+ *
+ * @param {string} id
+ *
+ * @returns {string[]}
+ */
+function idToReferenceFormatOptions(id) {
+  return ['', '\'', '"'].map(function(quoteSymbol) {
+    return idToReference(id, quoteSymbol);
+  });
 }

